# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# ilbe123 <a3057016@drdrb.net>, 2014
# Chris Park <utopinator@gmail.com>, 2015-2016
# snotree <cknblue@gmail.com>, 2017
# testsubject67 <deborinha97@hotmail.com>, 2014
# eukim <eukim@redhat.com>, 2009
# Dr.what <javrick6@naver.com>, 2014
# 류종헌, 2015
# sangmin lee <sangminlee7648@gmail.com>, 2016
# Sungjin Kang <potopro@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-09-20 18:02+0200\n"
"PO-Revision-Date: 2017-09-21 01:33+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Korean (http://www.transifex.com/otf/torproject/language/ko/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ko\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: ../tails_installer/creator.py:101
msgid "You must run this application as root"
msgstr "루트에서 이 응용프로그램을 실행해야 합니다."

#: ../tails_installer/creator.py:147
msgid "Extracting live image to the target device..."
msgstr "대상 장치에 라이브 이미지를 추출 중..."

#: ../tails_installer/creator.py:154
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "%(speed)d MB/초로 장치에 기록되었습니다"

#: ../tails_installer/creator.py:184
msgid "Setting up OLPC boot file..."
msgstr "OLPC 부팅 파일을 설정합니다 ..."

#: ../tails_installer/creator.py:321
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "다음 명령의 실행에 문제가있었습니다 : `%(command)s`.\n자세한 오류 로그 '%(filename)s'에 적혀 있습니다."

#: ../tails_installer/creator.py:340
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "LiveCD 이미지 SHA1 체크섬 확인 중 ..."

#: ../tails_installer/creator.py:344
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "LiveCD 이미지 SHA256 체크섬 확인 중 ..."

#: ../tails_installer/creator.py:360
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "오류: Live CD의 SHA1이 잘못되어 있습니다. 이러한 프로그램을 --noverify 인수와 함께 실행하여 유효성 검사를 해결할 수 있습니다. "

#: ../tails_installer/creator.py:366
msgid "Unknown ISO, skipping checksum verification"
msgstr "알 수 없는 ISO 체크섬 검증을 생략합니다"

#: ../tails_installer/creator.py:377
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "디바이스에 여유공간이 없습니다.\n%dMB ISO + %dMB 이상 > %dMB 여유 공간"

#: ../tails_installer/creator.py:384
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "%sMB 퍼시스턴트 구역을 작성 중"

#: ../tails_installer/creator.py:445
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "%(infile)s를 %(outfile)s에 복사 할 수 없습니다 : %(message)s"

#: ../tails_installer/creator.py:459
msgid "Removing existing Live OS"
msgstr "기존 LiveOS을 삭제 중"

#: ../tails_installer/creator.py:468 ../tails_installer/creator.py:479
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "%(file)s를 chmod할 수 없습니다: %(message)s"

#: ../tails_installer/creator.py:472
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "이전 LiveOS에서 파일을 삭제할 수 없습니다 : %(message)s"

#: ../tails_installer/creator.py:484
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "이전 LiveOS에서 디렉토리를 삭제할 수 없습니다 : %(message)s"

#: ../tails_installer/creator.py:532
#, python-format
msgid "Cannot find device %s"
msgstr "%s 장치를 찾을 수 없음 "

#: ../tails_installer/creator.py:701
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "%(device)s에 쓸 수 없습니다. 스킵합니다."

#: ../tails_installer/creator.py:725
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "대상 장치 %(device)s 파티션의 일부가 마운트되었습니다. 설치 프로세스가 시작하기 전에 이들은 마운트 해제됩니다."

#: ../tails_installer/creator.py:768 ../tails_installer/creator.py:991
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "알 수없는 파일 시스템. 장치를 다시 포맷해야할지도 모릅니다."

#: ../tails_installer/creator.py:771 ../tails_installer/creator.py:994
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "지원되지 않는 파일 시스템: %s "

#: ../tails_installer/creator.py:788
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr "장치를 탑재하는 동안 알 수 없는 GLib 예외가 발생했습니다:%(message)s"

#: ../tails_installer/creator.py:792
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "장치를 마운트 할 수 없습니다: %(message)s"

#: ../tails_installer/creator.py:797
msgid "No mount points found"
msgstr "마운트 포인트를 찾을 수 없습니다."

#: ../tails_installer/creator.py:808
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "'%(device)s'에 unmount_device를 입력 중"

#: ../tails_installer/creator.py:818
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "%(device)s에 마운트 된 파일 시스템을 마운트 해제 중"

#: ../tails_installer/creator.py:822
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "%(device)s에 '%(udi)s'를 분리 중"

#: ../tails_installer/creator.py:832
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "마운트 해제 후 %s의 실체를 마운트"

#: ../tails_installer/creator.py:845
#, python-format
msgid "Partitioning device %(device)s"
msgstr "장치 %(device)s의 파티션을 구성 중"

#: ../tails_installer/creator.py:920
#, python-format
msgid "Updating properties of system partition %(system_partition)s"
msgstr "%(system_partition)s 시스템 파티션의 속성을 업데이트합니다."

#: ../tails_installer/creator.py:976
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "지원되지 않는 장치 '%(device)s', 버그를 보고하십시오."

#: ../tails_installer/creator.py:979
msgid "Trying to continue anyway."
msgstr "어쨌든간에 계속하기."

#: ../tails_installer/creator.py:988 ../tails_installer/creator.py:1381
msgid "Verifying filesystem..."
msgstr "파일 시스템 확인 중... "

#: ../tails_installer/creator.py:1012
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "볼륨 레이블을 변경할 수 없습니다: %(message)s"

#: ../tails_installer/creator.py:1017 ../tails_installer/creator.py:1414
msgid "Installing bootloader..."
msgstr "부트 로더를 설치 중..."

#: ../tails_installer/creator.py:1043
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr "'%s' COM32 모듈을 찾을 수 없습니다."

#: ../tails_installer/creator.py:1051 ../tails_installer/creator.py:1432
#, python-format
msgid "Removing %(file)s"
msgstr "%(file)s를 삭제 중"

#: ../tails_installer/creator.py:1163
#, python-format
msgid "%s already bootable"
msgstr "%s는 이미 부팅 가능합니다"

#: ../tails_installer/creator.py:1183
msgid "Unable to find partition"
msgstr "파티션을 찾을 수 없습니다."

#: ../tails_installer/creator.py:1206
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "%(device)s를 FAT32로 포맷 중"

#: ../tails_installer/creator.py:1266
msgid "Could not find syslinux' gptmbr.bin"
msgstr "syslinux' gptmbr.bin을 찾을 수 없습니다."

#: ../tails_installer/creator.py:1279
#, python-format
msgid "Reading extracted MBR from %s"
msgstr "%s에서 추출한 MBR 읽기"

#: ../tails_installer/creator.py:1283
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr "%(path)s에서 추출 된 MBR을 읽을 수 없습니다."

#: ../tails_installer/creator.py:1296 ../tails_installer/creator.py:1297
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "%s의 마스터 부트 레코드를 재설정 중"

#: ../tails_installer/creator.py:1302
msgid "Drive is a loopback, skipping MBR reset"
msgstr "드라이브가 루프백입니다. MBR 재설정을 건너 뜁니다."

#: ../tails_installer/creator.py:1306 ../tails_installer/creator.py:1560
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "%s의 SHA1 계산 중"

#: ../tails_installer/creator.py:1331
msgid "Synchronizing data on disk..."
msgstr "디스크의 데이터를 동기화 중..."

#: ../tails_installer/creator.py:1373
msgid "Error probing device"
msgstr "장치를 엄격하게 조사하는 동안 오류"

#: ../tails_installer/creator.py:1375
msgid "Unable to find any supported device"
msgstr "지원되는 장치를 찾을 수 없습니다."

#: ../tails_installer/creator.py:1385
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "USB 키가 연결되어 FAT 파일 시스템으로 포맷되어 있는지 확인 "

#: ../tails_installer/creator.py:1388
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "지원되지 않는 파일 시스템: %s\nUSB 키를 FAT 파일 시스템으로 백업 및 포맷하십시오.  "

#: ../tails_installer/creator.py:1455
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Win32_LogicalDisk를 사용할 수 없습니다. win32com 쿼리가 어떠한 결과도 반환하지 않았습니다."

#: ../tails_installer/creator.py:1508
msgid "Cannot find"
msgstr "찾을 수 없음"

#: ../tails_installer/creator.py:1509
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr "이 프로그램을 실행하기 전에 전체 tails-installer 압축 파일을 추출 해야 합니다."

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "알 수 없는 자료: %s  "

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "%s 다운로드 중...  "

#: ../tails_installer/gui.py:209
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "오류: 레이블을 설정할 수 없거나 장치의 UUID를 검색할 수 없습니다. 계속 진행할 수 없습니다.  "

#: ../tails_installer/gui.py:258
#, python-format
msgid "Installation complete! (%s)"
msgstr "설치 완료! (%s)"

#: ../tails_installer/gui.py:263
msgid "Tails installation failed!"
msgstr "Tails 설치가 실패했습니다!"

#: ../tails_installer/gui.py:363
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "경고 : 이 프로그램을 실행하기 위해 관리자 권한으로 전환해야 합니다. 아이콘을 오른쪽 마우스로 클릭하여 속성으로 들어간 후, 호환성 탭에서 \"관리자 권한으로 프로그램 실행\"을 체크해 주세요."

#: ../tails_installer/gui.py:375
msgid "Tails Installer"
msgstr "Tails 설치 프로그램"

#: ../tails_installer/gui.py:454
msgid "Upgrade"
msgstr "업그레이드"

#: ../tails_installer/gui.py:458 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr "설치"

#: ../tails_installer/gui.py:465
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:476
msgid "No ISO image selected"
msgstr ""

#: ../tails_installer/gui.py:477
msgid "Please select a Tails ISO image."
msgstr ""

#: ../tails_installer/gui.py:516
msgid "No device suitable to install Tails could be found"
msgstr "Tails 설치에 적합한 장치를 찾을 수 없음"

#: ../tails_installer/gui.py:518
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr "최소 %0.1fGB의 USB 플래시 드라이브 또는 SD 카드를 연결하십시오."

#: ../tails_installer/gui.py:552
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr "USB 스틱 \"%(pretty_name)s\"는 제조업체가 제거 할 수 없도록 구성하였기 때문에 Tails가 시작되지 않습니다. 다른 모델로 설치를 시도해 보십시오."

#: ../tails_installer/gui.py:562
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "선택된 장치 \"%(pretty_name)s\" 의 용량이 모자라서 Tail를 설치할 수 없습니다 (최소 %(size)s GB 필요) "

#: ../tails_installer/gui.py:584
msgid "An error happened while installing Tails"
msgstr "Tails 설치 중 오류가 발생했습니다."

#: ../tails_installer/gui.py:596
msgid "Refreshing releases..."
msgstr "자료 업데이트 중..."

#: ../tails_installer/gui.py:601
msgid "Releases updated!"
msgstr "자료가 업데이트되었습니다!"

#: ../tails_installer/gui.py:643
msgid "Installation complete!"
msgstr "다운로드 완료! "

#: ../tails_installer/gui.py:644
msgid "Installation was completed."
msgstr "설치가 완료되었습니다."

#: ../tails_installer/gui.py:693
msgid "Unable to mount device"
msgstr "장치를 구성할 수 없습니다."

#: ../tails_installer/gui.py:700 ../tails_installer/gui.py:730
msgid "Confirm the target USB stick"
msgstr ""

#: ../tails_installer/gui.py:701
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr ""

#: ../tails_installer/gui.py:717
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:725
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr ""

#: ../tails_installer/gui.py:726
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr ""

#: ../tails_installer/gui.py:776
msgid "Download complete!"
msgstr "다운로드 완료! "

#: ../tails_installer/gui.py:780
msgid "Download failed: "
msgstr "다운로드 실패:  "

#: ../tails_installer/gui.py:781
msgid "You can try again to resume your download"
msgstr "다운로드를 다시 시작할 수 있습니다  "

#: ../tails_installer/gui.py:789
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "선택된 파일은 읽을 수 없습니다. 권한을 수정하거나 다른 파일을 선택하십시오."

#: ../tails_installer/gui.py:795
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "지정된 파일을 사용할 수 없으니 루트 디렉토리에 ISO 파일을 이동시키십시오. (예 : C:\\)"

#: ../tails_installer/gui.py:801
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s 선택됨"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr "ISO에서 LiveOS를 찾을 수 없습니다."

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr "기본 블럭 장치를 추측 할 수 없습니다: %s"

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr "`%s`를 실행하는 중에 문제가 발생했습니다.\n%s\n%s"

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr "'%s' 는 존재하지 않습니다."

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr "'%s' 는 디렉토리가 아닙니다"

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr "'%(filename)s' 건너뛰기"

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr "`%s` 를 실행하는 중에 문제가 발생했습니다.%s\n%s"

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr "쓰기를 위해 기기를 열 수 없습니다."

#: ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr ""

#: ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr ""

#: ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr ""

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr "다운로드 받을 배포처를 선택:"

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr ""

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr ""
